contract/src/airdrop/prepare.js

```diff
@@ -0,0 +1,267 @@
+
+          this.state.currentEpochEndTime =
+            absTime + newEpochDetails.windowLength;
+        
+          this.state.earlyClaimBonus =
+            this.facets.helper.getDistributionEpochDetails().tokenQuantity;
+
+           this.facets.helper.updateDistributionMultiplier(newEpochDetails);
+        },
+        async updateDistributionMultiplier(newEpochDetails) {
+          const {
+            facets,
+          } = this;
+          const epochDetails = newEpochDetails;
+
+          const { absValue } = await E(timer).getCurrentTimestamp();
+          this.state.currentCancelToken = cancelTokenMaker();
+
+          void E(timer).setWakeup(
+            TimeMath.absValue(absValue + epochDetails.windowLength),
+            makeWaker(
+              'updateDistributionEpochWaker',
+              ({ absValue: latestTsValue }) => {
+                this.state.currentEpoch += 1;
+                facets.helper.updateEpochDetails(
+                  latestTsValue,
+                  this.state.currentEpoch,
+                );
+              },
+            ),
+          );
+          return 'wake up successfully set.';
+        },
+      },
+      creator: {
+        createPayment() {
+          return this.state.internalPurse.withdraw(
+            this.facets.helper.makeAmountForClaimer(),
+          );
+        },
+        async prepareAirdropCampaign() {
+      
+          this.state.currentCancelToken = cancelTokenMaker();
+          const {
+            facets,
+            state: {
+              dsm: { transitionTo },
+            },
+          } = this;
+          console.groupEnd();
+          await E(timer).setWakeup(
+            TimeMath.absValue(startTime),
+            makeWaker('claimWindowOpenWaker', ({ absValue }) => {
+              transitionTo(states.OPEN);
+              facets.helper.updateEpochDetails(
+                absValue,
+                this.state.currentEpoch,
+              );
+            }),
+            this.state.cancelToken,
+          );
+        },
+      },
+      claimer: {
+        getStatus() {
+          return this.state.dsm.getStatus();
+        },
+        view() {
+          const { count } = this.state;
+          return count;
+        },
+        claim() {
+          assert(
+            this.facets.claimer.getStatus() === states.OPEN,
+            'Claim attempt failed.',
+          );
+          const airdropPayment = this.facets.creator.createPayment();
+
+          /**
+           * @param payment
+           */
+          const claimHandler = payment => 
+          /**
+           * @param {UserSeat} seat
+           * @param offerArgs
+           */
+          async (seat, offerArgs) => {
+            // const payoutPurse = createPurse(tokenIssuer);
+
+            // payoutPurse.deposit(payment);
+
+            seat.exit();
+            return harden({
+              message: 'Here is your payout purse - enjoy!',
+              airdrop: payment,
+            });
```

Comment:
I think you want to use `depositToSeat` to give out the payment.

The smart wallet doesn't grok payments in this way.